python/py-kraken/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kraken
+version             4.2.0
+license             Apache
```

Comment:
should be `Apache-2`

Reply:
additionally: is this pure Python code, if so one should probably add `supported_archs noarch`

Reply:
Thanks done.